examples/widgets/histogram.html

```diff
@@ -0,0 +1,98 @@
+
+  <link rel="stylesheet" type="text/css" href="https://libs.cartocdn.com/airship-style/v2.3/airship.css" />
+
+  <style>
+    body {
+      margin: 0;
+      padding: 0;
+    }
+
+    #map {
+      width: 100vw;
+      height: 100vh;
+    }
+
+    #panel {
+      width: 300px;
+    }
+
+    #panel .as-box {
+      width: 250px;
+    }
+  </style>
+</head>
+
+<body class="as-app-body as-app">
+  <div class="as-content">
+    <main class="as-main">
+      <div class="as-map-area">
+        <!-- map -->
+        <div id="map"></div>
+
+        <!-- panel -->
+        <div id="panel" class="as-panel as-panel--top as-panel--right as-bg--ui-01">
+          <div class="as-panel__element as-p--16 as-body">
+            <div class="as-container as-container--scrollable">
+              <section class="as-box">
+                <h1 class="as-title">Histogram widget</h1>
+                <!-- <h4 class="as-subheader as-mb--12">
+                  Create a widget, aggregating values per category, for those features currently
+                  visible. It includes a filtering feature out of the box
+                </h4> -->
+              </section>
+              <section class="as-box">
+                <as-histogram-widget class="local" heading="Local Estimated Population" description="Distribution of estimated population in countries" show-clear>
+                </as-histogram-widget>
+
+                <as-histogram-widget class="remote" heading="Remote Estimated Population" description="Distribution of estimated population in countries">
+                </as-histogram-widget>
+              </section>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+  </div>
+
+  <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js"></script>
+  <script src="https://unpkg.com/deck.gl@8.2.0-beta.1/dist.min.js"></script>
+  <script src="https://libs.cartocdn.com/airship-components/v2.3/airship.js"></script>
+  <script src="/dist/umd/index.min.js"></script>
+
+  <script>
+    const widgetElement = document.querySelector('as-category-widget');
+
+    async function initialize() {
+      carto.setDefaultCredentials({ username: 'public' });
+
+      const deckMap = carto.viz.createMap();
+
+      const countriesLayer = new carto.viz.Layer('ne_50m_admin_0_countries');
+      await countriesLayer.addTo(deckMap);
+
+      const localDataview = new carto.dataview.Histogram(countriesLayer, 'pop_est', {
+        bins: 10, start: 1000000, end: 127078679, mode: 'REMOTE_FILTERED'
+      });
+      const localHistogramWidget = new carto.widget.Histogram(document.querySelector('.local'), localDataview);
+
+
+      const remoteDataview = new carto.dataview.Histogram(countriesLayer, 'pop_est', {
+        bins: 10, start: 1000000, end: 127078679, mode: 'LOCAL'
+      });
+      const remoteHistogramWidget = new carto.widget.Histogram(document.querySelector('.remote'), remoteDataview);
+    }
```

Comment:
I find this naming a bit confusing. Shouldn't the first `localDataview` be called something in the line of `viewportHistogramDataview` (& viewportHistogramWidget)?. 

Also. using 'LOCAL' for the `remoteDataview` feels bizarre

Reply:
Yes, I didn't review this before committing. I'm gonna change this.